topics/covid19/current-patterns-transmission/_posts/2020-03-22-ICU-projections.md

```diff
@@ -125,9 +110,30 @@ Assuming the proportion of COVID-19 cases needing critical care remains constant
 <br>
 
 
+## Regional Results {.tabset .tabset-fade .tabset-pills}
+
+### NHS regions (excl. London)
+
+For the analysis excluding London, the starting point for admissions projections is taken from the *18th March 2020*.
```

Comment:
Could you add a sentence right after along the lines of:

Average demand ranging from xxx (CI95%: xxx ; xxx) to xxx (CI95%: xxx ; xxx) critical care beds every day on the 31st March 2020.